packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -509,13 +537,26 @@ export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState
 
```

Comment:
It seems strange that this calls onMenuClick... If this want to call onMenuClick, why isn't this calling _onMenuClick?

Reply:
We want to trigger the primary action when we hit the enter button if we're focusing on the html element. This works for buttons, unfortunately, the container is a div so pressing enter won't trigger the onClick event. 

I can move the code in onMenuClick and call it in both places, but they would both also reference onClick events.

Reply:
I think as long as _onToggleMenu is updating the focus correctly this is fine

Reply:
This is resolved, we'll keep this code here and add the focus change to onToggleMenu